testing/i3wm-gaps/APKBUILD

```diff
@@ -0,0 +1,41 @@
+# Maintainer: Jakub Skrzypnik <j.skrzypnik@openmailbox.org>
+pkgname=i3wm-gaps
+pkgver=4.13
+pkgrel=0
+pkgdesc="A tiling window manager with gaps and other additional features"
+url="https://github.com/Airblader/i3"
+arch="all"
+license="BSD"
+replaces="i3wm"
+makedepends="bison flex libxcb-dev xcb-util-cursor-dev
+             xcb-util-keysyms-dev xcb-util-wm-dev libev-dev pango-dev cairo-dev
+             yajl-dev startup-notification-dev pcre-dev libxkbcommon-dev
+             xcb-util-xrm-dev asciidoc perl xmlto autoconf automake libtool"
+subpackages="$pkgname-doc"
+source="i3wm-$pkgver.tar.gz::https://github.com/Airblader/i3/archive/$pkgver.tar.gz
+        musl.patch"
+
+builddir="$srcdir/i3-$pkgver"
+
+build() {
+  cd "$srcdir/i3-$pkgver"
+  autoreconf -fi || return 1
+  mkdir -p "$builddir"/build && cd "$builddir"/build
+  ../configure --prefix="/usr"
+  make || return 1
+}
+
+package() {
+  cd "$builddir"/build
+  make DESTDIR="$pkgdir/" install || return 1
+
+  install -d "$pkgdir/usr/share/man/man1"
+  install -m644 man/*.1 "$pkgdir"/usr/share/man/man1/ || return 1
+}
+
+md5sums="a591216c8a84d27431b057e4d01b151e  i3wm-4.13.tar.gz
+aa998beb0b9a83d2910508768ac7faae  musl.patch"
+sha256sums="e80c07e525a07f2d3d2f02d282774d3264b6d1f165713c9903c06626ba4d2571  i3wm-4.13.tar.gz
+b7d084c53addc71fda13ccb8b3dcca2a32d9ca1590a28bf3be1a0c61870d3817  musl.patch"
+sha512sums="9b0c888c1c412622282f5e56ba6763eba3af88682d1652d1a1e6ce4e7fcdbec30e4fd6731b7e4d306bd33871a2581e0979714b47c9614210302c491ce9366ba8  i3wm-4.13.tar.gz
+8ce7d00371c43b93dabbe0dadf9caf7c58a68f4a0079f5a9b9552c15c55bfa0df16d7e87a281595af2ac5254632ba28ccf82a467cea16159b41490f6f2910299  musl.patch"
```

Comment:
Please use only tabs for indentation (and alignment) in APKBUILDs and other shell scripts.